packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -111,6 +125,148 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     }
   }
 
+  componentDidMount() {
+    if (this.props.useWindowMoveEvents) {
+      window.addEventListener('mouseup', this.handleMouseUp);
+      window.addEventListener('mousemove', this.handleMouseMove);
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.props.useWindowMoveEvents) {
+      window.removeEventListener('mouseup', this.handleMouseUp);
+      window.removeEventListener('mousemove', this.handleMouseMove);
+    }
+  }
+
+  handleMouseUp = () => {
+    const { useWindowMoveEvents } = this.props;
+    const { brushingType } = this.state;
+
+    if (
+      useWindowMoveEvents &&
+      ['top', 'bottom', 'left', 'right', 'select', 'move'].includes(brushingType ?? '')
+    ) {
+      this.updateBrush((prevBrush: BaseBrushState) => {
+        const { start, end, extent } = prevBrush;
+
+        start.x = Math.min(extent.x0, extent.x1);
+        start.y = Math.min(extent.y0, extent.y0);
+        end.x = Math.max(extent.x0, extent.x1);
+        end.y = Math.max(extent.y0, extent.y1);
+
+        return {
+          ...prevBrush,
+          activeHandle: null,
+          isBrushing: false,
+          brushingType: undefined,
+        };
+      });
+    }
+  };
+
+  handleMouseMove = (event: MouseEvent) => {
```

Comment:
similar comment here about scoping to window handlers, and I think ideally we'd use `MouseTouchOrPointerEvent` (see example of this type being imported from `@visx/drag` in the brush utils file).  Being careful about mouse-specifics because of #845. 

```suggestion
  handleWindowPointerMove = (event: MouseTouchOrPointerEvent) => {
```